modules/manually-create-identity-access-management.adoc

```diff
@@ -55,28 +54,8 @@ $ openshift-install create manifests --dir=<installation_directory> <1>
 ----
 <1> For `<installation_directory>`, specify the directory name to store the
 files that the installation program creates.
-ifndef::aws[]
-//ConfigMap method is verified supported for Azure and GCP. This step can be removed when credentialsMode=Manual support is verified for these platforms.
-. Insert a config map into the manifests directory so that the Cloud Credential
-Operator is placed in manual mode:
-+
-[source,terminal]
-----
-$ cat <<EOF > mycluster/manifests/cco-configmap.yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cloud-credential-operator-config
-  namespace: openshift-cloud-credential-operator
-  annotations:
-    release.openshift.io/create-only: "true"
-data:
-  disabled: "true"
-EOF
-----
-endif::aws[]
-. Remove the `admin` credential secret created using your local cloud credentials.
-This removal prevents your `admin` credential from being stored in the cluster:
+
+. Remove the administrator-level credential secret created using your local cloud credentials. This removal prevents your administrator-level credential from being stored in the cluster:
```

Comment:
Ever since https://github.com/openshift/installer/pull/4416 merged, we no longer generate this Secret when the mode is set to "Manual".